src/components/ConfirmDelete.jsx

```diff
@@ -22,6 +22,7 @@ export default function ConfirmDelete({
   errorTitleId = 'SERVER_ERROR',
   error,
   titleId,
+  severity = 'error',
```

Comment:
What do you think of `alertSeverity` instead?

Reply:
Done.